test/spec/reverse_route_spec.coffee

```diff
@@ -0,0 +1,39 @@
+ReverseRoute = require '../../src/reverse_route'
+
+
+describe 'ReverseRoute', ->
+
+  beforeEach ->
+    @route = new ReverseRoute
+      path: '/some/path'
+      method: 'GET'
+
+  it 'exposes path', ->
+    expect(@route.path).to.equal '/some/path'
+
+  it 'exposes method', ->
+    expect(@route.method).to.equal 'GET'
+
+
+  describe 'behaves like a string', ->
```

Comment:
I don't see that "behaving like a string" is a desired behavior for a reversed route.